packages/sdks/snippets/react/src/components/CustomTabs.tsx

```diff
@@ -0,0 +1,59 @@
+import { RegisteredComponent } from "../../../../output/react/types/context/types";
```

Comment:
this need not be a component.tsx file, I think we can name it as `registered-components.ts` or `custom-components.ts` or even `builder-custom-components.ts`